tembo-pgmq-python/README.md

```diff
@@ -1,242 +1,290 @@
-# Tembo's Python Client for PGMQ
-
-## Installation
-
-Install with `pip` from pypi.org:
-
-```bash
-pip install tembo-pgmq-python
-```
-
-In order to use async version install with the optional dependecies:
-
-``` bash
-pip install tembo-pgmq-python[async]
-```
-
-
-Dependencies:
-
-Postgres running the [Tembo PGMQ extension](https://github.com/tembo-io/tembo/tree/main/pgmq).
-
-## Usage
-
-### Start a Postgres Instance with the Tembo extension installed
-
-```bash
-docker run -d --name postgres -e POSTGRES_PASSWORD=postgres -p 5432:5432 quay.io/tembo/pg16-pgmq:latest
-```
-
-### Using Environment Variables
-
-Set environment variables:
-
-```bash
-export PG_HOST=127.0.0.1
-export PG_PORT=5432
-export PG_USERNAME=postgres
-export PG_PASSWORD=postgres
-export PG_DATABASE=test_db
-```
-
-Initialize a connection to Postgres using environment variables:
+ # Tembo's Python Client for PGMQ
```

Comment:
@tavallaie - what happened with the formatting on this readme? It looks like every line got pre-pended with a space.
```suggestion
# Tembo's Python Client for PGMQ
```